README.md

```diff
@@ -168,6 +186,10 @@ Below are some commonly encountered issues and possible ways to resolve them. If
 - The app gets stuck on the Projects screen -- projects never load
   > Make sure the API is running on your local machine, and that your **api/.env** and **app/.env** files are configured correctly (see [Setup and first run](#setup-and-first-run) above)
 
+  > Check the URL for both `API_TUNNEL_SUBDOMAIN` and `STA_API_BASE_URL`. `STA_API_BASE_URL` should be prefixed with `https://` but not `API_TUNNEL_SUBDOMAIN`. 
```

Comment:
As mentioned above, this should also not include `.loca.lt`

Reply:
I have removed the .local.lt from API_TUNNEL_SUBDOMAIN. Forgive me, but should STA_API_BASE_URL also not include .local.lt? I assumed it did as was going on the example earlier on in the Readme around Step 8. To add to my confusion (and due to ignorance) my Expo setup works without the need for either. Does this mean .local.lt is not required on either? If so i will update this and remove from the STA_API_BASE_URL example!

Reply:
Sorry if I was unclear.

`API_TUNNEL_SUBDOMAIN` should never include `https://` or `.loca.lt` (e.g. `nadia-bloggs-abc123def456ghi789`)

`STA_API_BASE_URL` should be a full URL (e.g. `https://nadia-bloggs-abc123def456ghi789.loca.lt`)

Reply:
Thanks for the clarification!